src/cloudimagedirectory/transform/transform.py

```diff
@@ -291,13 +280,15 @@ def run(self, data: type[Transformer]) -> list:
 
         results.sort()
 
-        return [connection.DataEntry("v1/idx/list/image-names", results)]
+        # TODO: DataEntry expects a dictionary as its second argument, but this is
+        # passing a list. Should we update DataEntry to accept a list?
```

Comment:
Actually it should accept everything that can be serialized to JSON. Should it then be of type `any` or is there a type in Python like `serializable`? :)

Reply:
Ah, I was able to adjust it to be `list | dict | None`, which isn't great, but it gets us over this problem for now.